lib/stats/DefaultStatsPrinterPlugin.js

```diff
@@ -57,12 +59,14 @@ const SIMPLE_PRINTERS = {
 			: undefined,
 	"compilation.time": (time, { bold }) => `Time: ${bold(time)}ms`,
 	"compilation.builtAt": (builtAt, { bold }) => {
-		const builtAtDate = new Date(builtAt);
-		return `Built at: ${builtAtDate.toLocaleDateString(undefined, {
-			day: "2-digit",
-			month: "2-digit",
-			year: "numeric"
-		})} ${bold(builtAtDate.toLocaleTimeString())}`;
+		const d = new Date(builtAt);
+		const date = `${twoDigit(d.getMonth() + 1)}/${twoDigit(
+			d.getDate()
+		)}/${d.getFullYear()}`;
```

Comment:
Switching to ISO format would make the date easier to process for non-US users. Maybe something like `yyyy-mm-dd`.